schema/partner_show.js

```diff
@@ -105,7 +106,16 @@ const PartnerShowType = new GraphQLObjectType({
     },
     cover_image: {
       type: Image.type,
-      resolve: ({ image_versions, image_url }) => Image.resolve({ image_versions, image_url }),
+      resolve: ({ id, partner, image_versions, image_url }) => {
+        if (image_versions && image_versions.length && image_url) {
+          return Image.resolve({ image_versions, image_url });
+        }
```

Comment:
This should just be handled by the [image normalization](https://github.com/artsy/metaphysics/blob/master/schema/image/normalize.js) which is [https://github.com/artsy/metaphysics/blob/a68f56e4f39108957d5ec66b8832f49caaa26fa7/schema/image/index.js#L77](what that resolve is anyway)